configs/biggan/metafile.yml

```diff
@@ -64,7 +63,7 @@ Models:
     Task: Conditional GANs
   Weights: https://download.openmmlab.com/mmgen/biggan/biggan-deep_imagenet1k_128x128_cvt_hugging-face_rgb_20210728_111659-099e96f9.pth
 - Config: https://github.com/open-mmlab/mmgeneration/blob/master/configs/_base_/models/biggan/biggan-deep_256x256_cvt_hugging-face_rgb.py
-  In Collection: BigGAN
+  In Collection: README
```

Comment:
Such behavior is not correct. Please further check it.

Reply:
It seems that `In Collection` in all metafiles are `README`.

Reply:
No! It shouldn't be like this. You can have a look at the master branch.